### ISSUE TITLE
List all the open source libraries that Monica uses

### ISSUE THREAD

**Issue by djaiss** [monicahq/monica#298]
It would be great (and polite) to list and give full credit on other people's work.

**Comment by erdmenchen** [comment]
Not only would it be great, but there might be some dependencies where the license makes thjs mandatory...

**Comment by djaiss** [comment]
Indeed. Hence the importance, especially since Monica is AGPL today.

**Comment by asbiin** [comment]
Where do you want to list this ? A file on github, a page on monicahq.com, something else ?

**Comment by djaiss** [comment]
@asbiin I think it should live in the docs folder, with a link pointing to it in `readme.md` saying we want to thank those libraries.
